dash/development/build_renderer.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import string
+import shutil
+import logging
+
+import coloredlogs
+import fire
```

Comment:
don't wanna write `argparse` options
fire is smart enough if we have a docstring (google style e.g.), it generates a CLI usage guide from that.